src/telemetry/telemetry.bif

```diff
@@ -3,7 +3,17 @@
 
 module Telemetry;
 
+enum MetricType %{
```

Comment:
Why is this duplicated here?

Reply:
This is for making the enum `MetricType` enum available to Zeek script land, too.

I'll follow the example of the enum class you provided elsewhere (thanks!), but seem that has the same pattern of being replicated in a bif file:
https://github.com/zeek/zeek/blob/master/src/broker/comm.bif#L66

Reply:
Duplicating code is rarely a good idea. I can't tell why https://github.com/zeek/zeek/blob/master/src/broker/comm.bif#L66 does this, but it looks very suspicious to me (I'll look into the Broker BIF and probably file a PR for that). As long as the enum type is in a header (and it is), the BIF should simply include the proper header and use it.

Reply:
@rsmmr actually, https://github.com/zeek/zeek/blob/master/src/broker/comm.bif#L66 looks like it's from the recent WebSocket additions. Do you remember details why the code duplicates an enum?

Reply:
Not sure what's duplicated here, the `enum` in the bif should be making this available as a script-level type. Is that happening elsewhere already?

Reply:
The `enum` itself is duplicated.

Once as a C-style enum in https://github.com/zeek/zeek/blob/master/src/broker/comm.bif#L66 and once as `enum class` in https://github.com/zeek/zeek/blob/master/src/broker/Manager.h#L83. Wouldn't it work to use the "proper" definition from `broker/Manager.h` and make that available to script land directly?

Reply:
The first `enum` declaration is part of the BiF language, not a C construct itself. The manager could `#include` that bif-generated header with the resulting enum, but that's not so nice to use, which is why, iirc, I added a separate one instead (maybe there was a technical issue as well, don't remember).